web/templates/presence_row.hbs

```diff
@@ -1,37 +1,40 @@
 <li data-user-id="{{user_id}}" class="user_sidebar_entry {{#if user_list_style.WITH_AVATAR}}with_avatar{{/if}} {{#if is_current_user}}user_sidebar_entry_me {{/if}} narrow-filter {{#if faded}} user-fade {{/if}}">
     <div class="selectable_sidebar_block">
         {{#unless user_list_style.WITH_AVATAR}}<span class="{{user_circle_class}} user_circle"></span>{{/unless}}
-        <a class="user-presence-link"
-          href="{{href}}"
-          data-user-id="{{user_id}}"
-          data-name="{{name}}">
+        <div class="user-presence-row" data-user-id="{{user_id}}" data-name="{{name}}">
```

Comment:
We should be a bit careful about how many HTML elements we put into this sidebar, just because they are made in a large loop. 

Is this div even useful? We have `data-user-id` on the `li` at the top level, and I'm not sure what `data-name` is for, but I could believe that's just stale.

Reply:
It was holding some structure, but I was able to remove it fairly easily. Thanks for bringing it up!